### DISCUSSION TITLE
Connect wastewater network elements as radio buttons (instead of checkboxes)

### DISCUSSION THREAD

**Discussion by rbovard** [QGEP/QGEP#755] [Ideas]
Why is the "Connect wastewater network elements" is displayed as checkboxes?

Shouldn't they be radio buttons?

**Comment by urskaufmann** [comment]
for connection reach - wastewater node: you are right, radio buttons would be better
for connection catchment area - node: there is multiple choice possible, so checkboxes are ok

**Comment by rbovard** [comment]
Thanks @urskaufmann, I wasn't aware of the catchment area connection.

So the idea is to display the form depending on the connection type:
- Reach connection: radio buttons
- Catchment area connection: checkboxes (as now)

What's the procedure next? Should I open an issue?

**Comment by urskaufmann** [comment]
For me it's not a priority. If you start with the connection-tool next to the end of the reach you want to connect, then you will never have to click on this buttons. But in fact it would be not a good idea to click on both buttons (I never tried)...

**Comment by rbovard** [comment]
It's not a blocker but I think it's not ideal for the UX, especially for new users.

I've opened https://github.com/QGEP/QGEP/issues/760
